platform-configs/msi-pro-z790-p.robot

```diff
@@ -1,347 +1,10 @@
 *** Settings ***
-Resource    ../os-config/ubuntu-credentials.robot
+Resource    include/msi-z690-common.robot
```

Comment:
@macpijan did you push that `include` directory? I can't find it here

Reply:
@mkopec let me see :smile:

Reply:
I think I skipped something as I have changed the name of this dir from `variables` to `include`. Commit might need some cleanup, but the idea should be already there

Reply:
Thanks, I'll try to follow with this idea for pcengines boards